packages/marshaller/src/ddl2/dashboard.ts

```diff
@@ -36,19 +31,25 @@ export class Dashboard extends DockPanel {
         return this;
     }
 
-    save(): IDashboardPersist {
+    //  Used to delay load a layout after a render...
+    private _layoutObj: object = null;
+    layoutObj(): object | null;
+    layoutObj(_: object | null): this;
+    layoutObj(_?: object): this | object | null {
```

Comment:
Shouldn't this `object` be a high priority candidate for having a well defined interface?

Reply:
Probably not - I must first take we a string serialization - its the phosphor layout not ours...